lib/recurly.js

```diff
@@ -66,6 +71,7 @@ export class Recurly extends Emitter {
       token: bind(this, bankAccount.token),
       bankInfo: bind(this, bankAccount.bankInfo)
     };
+    this.fraud = new Fraud(this);
```

Comment:
Yeah, I had a bit of trouble deciphering where the parent stuff was, etc. I almost put it in the `parent()` function here:

https://github.com/recurly/recurly-js/blob/9b7fcf68f538d8e012f0ed020ca7bf7d9793c20d/lib/recurly.js#L155-L179

But, since everything else in that function is hosted fields related, I wasn't sure if it was a good place.